tools/clang/test/HLSLFileCheck/hlsl/workgraph/common_input_parameters.hlsl

```diff
@@ -20,25 +20,25 @@
 
 
 // CHECK:  %[[node_ptr:.+]] = call %struct.RECORD.0 addrspace(6)* @dx.op.getNodeRecordPtr.struct.RECORD.0(i32 239, %dx.types.NodeRecordHandle %[[annotHdl]], i32 0)  ; GetNodeRecordPtr(recordhandle,arrayIndex)
-// CHECK:  %[[ptr:.+]] = getelementptr inbounds %struct.RECORD.0, %struct.RECORD.0 addrspace(6)* %[[node_ptr]], i32 0, i32 0, i32 0
+// CHECK:  %[[ptr:.+]] = getelementptr {{(inbounds )?}}%struct.RECORD.0, %struct.RECORD.0 addrspace(6)* %[[node_ptr]], i32 0, i32 0, i32 0
```

Comment:
Pretty sure changes in this file are unnecessary if you only do `MergeGepUse` after `ReplaceUsesForLoweredUDTImpl` instead of before.